graph/TypeGraph.java

```diff
@@ -297,14 +298,15 @@ public <ORDER extends Order> Forwardable<TypeVertex, ORDER> roleTypes(ORDER orde
         return iterateSorted(getSubtypes(rootRoleType()), order);
     }
 
-    private NavigableSet<TypeVertex> ownedAttributeTypes(TypeVertex owner, boolean isKey) {
+    private NavigableSet<TypeVertex> ownedAttributeTypesImpl(TypeVertex owner, Set<TypeQLToken.Annotation> annotations) {
```

Comment:
let's use `fetchOwnedAttributeTypes` instead of `...Impl`

Also put it below the caller